infra/terragrunt.hcl

```diff
@@ -115,4 +119,6 @@ remote_state {
 inputs = merge(
   local.account_vars.locals,
   local.environment_vars.locals,
-)
\ No newline at end of file
+)
+
+# If Atlantis is unavailable, use local deployment by commenting out line 101 and profile on line 58 whilst commemtiing out references to role_arn line 102 and 65-68
```

Comment:
Would not be better to add this commend above the lines 101 and 58?
Something like "If Atlantis is unavailable please uncomment line below"